plugins/CoreHome/templates/_headerMessage.twig

```diff
@@ -8,12 +8,14 @@
 {% endset %}
 
 {% if
-    (latest_version_available and not isPiwikDemo and hasSomeViewAccess and not isUserIsAnonymous and showUpdateNotificationToUser)
-    or (
-        isSuperUser
-        and (
-            (isManualUpdateCheck is defined and isManualUpdateCheck and lastUpdateCheckFailed is defined and lastUpdateCheckFailed)
-            or (isAdminArea is defined and isAdminArea)
+    isAutoUpdateEnabled and (
```

Comment:
Should we also open a PR for Cloud to remove the no longer necessary `display: none` on their side? Or just ping them?

Reply:
I thought about it, we can just ping them.

Reply:
ref. PG-3097